File: trunking/v1/origination_url/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Trunking.V1.Trunk;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var originationUrl = OriginationUrlResource.Create(
            weight: 1,
            priority: 1,
            enabled: true,
            friendlyName: "FriendlyName",
            sipUrl: new Uri("https://example.com"),
            pathTrunkSid: "TRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(originationUrl.Sid);
    }
}

File: trunking/v1/origination_url/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

origination_url = @client.trunking
                         .trunks('TRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                         .origination_urls
                         .create(
                            weight: 1,
                            priority: 1,
                            enabled: true,
                            friendly_name: 'friendly_name',
                            sip_url: 'https://example.com'
                          )

puts origination_url.sid
